docs/src/main/sphinx/sql/show-role-grants.rst

```diff
@@ -12,5 +12,5 @@ Synopsis
 Description
 -----------
 
-List non-recursively the ``ROLE``s that have been granted to the session user
-in ``catalog``, or the current catalog if ``catalog`` is not specified.
+List non-recursively the roles that have been granted to the current user in the specified ``catalog``,
```

Comment:
"the specified catalog" without ``` `` ```?

cc @mosabua

Reply:
sure .. its not a code snippet..